include/inviwo/core/network/networkedge.h

```diff
@@ -44,15 +44,13 @@ class ProcessorNetwork;
  * Represents a edge in the ProcesorNetwork, either a PortConnection or a PropertyLink, as a
  * pair of paths, i.e. dot separated identifiers.
  */
-struct IVW_CORE_API NetworkEdge : Serializable {
+struct IVW_CORE_API NetworkEdge {
```

Comment:
Why? The functions are still there…

Reply:
It is enough to have the functions to be serializeable now, there is no need to also inherit the base class. This avoids some vtables and stuff, don't think it matters so much in this case.